src/browser/modules/Sidebar/About.jsx

```diff
@@ -18,6 +18,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import { version } from '../../../../package.json'
```

Comment:
Good idea. 
Maybe we should update the version in the `pom.xml` file as well when doing this?